Dockerfiles/scid/install_packages.R

```diff
@@ -0,0 +1,9 @@
+withCallingHandlers({
+  install.packages("BiocManager", repos="https://cloud.r-project.org/")
+  BiocManager::install(version = "devel", ask = FALSE);
```

Comment:
Why are you installing from the devel version of BioConductor? Actual versioned releases are likely more stable.